### DISCUSSION TITLE
Ordering fuzzy search results

### DISCUSSION THREAD

**Discussion by nickchampion** [ravendb/ravendb#14787] [General]
Hi,

We are trying to find matching documents by name field in the database using fuzzy search, the results from which are not sorted by the closest match. What would your recommendation be for the best approach to finding documents with closely matching names to present users with a sorted lists with closest match at the top?

Thanks

**Comment by garayx** [comment]
Hi 

You can use `OrderByScore()`, see:
https://groups.google.com/g/ravendb/c/4dTzDjGqbcg/m/3yGN7TOsBgAJ
